earth_enterprise/src/fusion/autoingest/sysman/StateUpdater_unittest.cpp

```diff
@@ -977,6 +989,90 @@ TEST_F(StateUpdaterTest, Cancel) {
   assertStateNotSet(sm, "ci3");
 }
 
+TEST_F(StateUpdaterTest, FailedInput) {
+  SetVersions(sm, {
+                    MockVersion("l"),
```

Comment:
I get that one's an input and one's a listener, but `l` and `i` are kind of hard to tell apart.